target/ppc/fpu_helper.c

```diff
@@ -2939,10 +2986,8 @@ VSX_CVT_FP_TO_INT_VECTOR(xscvqpuwz, float128, uint32, f128, VsrD(0), 0x0ULL)
  *   ttp   - target type (float32 or float64)
  *   sfld  - source vsr_t field
  *   tfld  - target vsr_t field
- *   jdef  - definition of the j index (i or 2*i)
- *   sfprf - set FPRF
  */
```

Comment:
This RFC is still missing comments that should be added for a final version (here, it's missing the `fpscr_mask` documentation)